components/match2/wikis/mobilelegends/match_summary.lua

```diff
@@ -0,0 +1,328 @@
+	then
+		return nil
+	end
+
+	row	:addClass('brkts-popup-body-game')
+		:css('font-size', '85%')
+		:css('overflow', 'hidden')
+
+	row:addElement(CustomMatchSummary._opponentChampionsDisplay(championsData[1], _NUM_CHAMPIONS_PICK, false))
+	row:addElement(CustomMatchSummary._createCheckMark(game.winner == 1))
+	row:addElement(mw.html.create('div')
+		:addClass('brkts-popup-body-element-vertical-centered')
+		:wikitext(CustomMatchSummary._createAbbreviation{
+			title = String.isEmpty(game.length) and ('Game ' .. gameIndex .. ' picks') or 'Match Length',
+			text = game.length or ('Game ' .. gameIndex),
+		})
+	)
+	row:addElement(CustomMatchSummary._createCheckMark(game.winner == 2))
+	row:addElement(CustomMatchSummary._opponentChampionsDisplay(championsData[2], _NUM_CHAMPIONS_PICK, true))
+
+	-- Add Comment
+	if not Logic.isEmpty(game.comment) then
+		row:addElement(MatchSummary.Break():create())
+		local comment = mw.html.create('div')
+		comment :wikitext(game.comment)
+				:css('margin', 'auto')
+		row:addElement(comment)
+	end
+
+	return row
+end
+
+function CustomMatchSummary._createCheckMark(isWinner)
+	local container = mw.html.create('div')
+		:addClass('brkts-popup-spaced')
+		:css('line-height', '27px')
+		:css('margin-left', '3%')
+		:css('margin-right', '3%')
+
+	if isWinner then
+		container:node(_GREEN_CHECK)
+	else
+		container:node(_NO_CHECK)
+	end
+
+	return container
+end
+
+function CustomMatchSummary._createOpponent(opponent, side)
+	return OpponentDisplay.BlockOpponent{
+		flip = side == 'left',
+		opponent = opponent,
+		overflow = 'wrap',
+		teamStyle = 'short',
+	}
+end
+
+function CustomMatchSummary._createScore(opponent)
+	return OpponentDisplay.BlockScore{
+		isWinner = opponent.placement == 1 or opponent.advances,
+		scoreText = OpponentDisplay.InlineScore(opponent),
+	}
+end
+
+function CustomMatchSummary._createAbbreviation(args)
+	return '<i><abbr title="' .. args.title .. '">' .. args.text .. '</abbr></i>'
+end
+
+function CustomMatchSummary._opponentChampionsDisplay(opponentChampionsData, numberOfChampions, flip, isBan)
+	local opponentChampionsDisplay = {}
+	local color = opponentChampionsData.color or ''
+	opponentChampionsData.color = nil
+
+	for index = 1, numberOfChampions do
+		local champDisplay = mw.html.create('div')
+		:addClass('brkts-popup-side-color-' .. color)
+		:css('float', flip and 'right' or 'left')
+		:node(ChampionIcon._getImage{
+			champ = opponentChampionsData[index],
+			class = 'brkts-champion-icon',
+		})
+		if index == 1 then
+			champDisplay:css('padding-left', '2px')
+		elseif index == numberOfChampions then
+			champDisplay:css('padding-right', '2px')
+		end
+		table.insert(opponentChampionsDisplay, champDisplay)
+	end
+
+	if flip then
+		opponentChampionsDisplay = Array.reverse(opponentChampionsDisplay)
+	end
+
+	local display = mw.html.create('div')
+	if isBan then
+		display:addClass('brkts-popup-side-shade-out')
+		display:css('padding-' .. (flip and 'right' or 'left'), '4px')
```

Comment:
is this correct?

Reply:
i think so
could be done via css sheet though

Reply:
adjusted in combination with https://liquipedia.net/commons/index.php?title=MediaWiki%3ACommon.css%2FBrackets.css&type=revision&diff=386538&oldid=383666